frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -2013,10 +2013,18 @@ mod tests {
 	#[test]
 	fn untrusted_score_verification_is_respected() {
 		ExtBuilder::default().build_and_execute(|| {
+			use frame_election_provider_support::SequentialPhragmen;
+			use sp_npos_elections::ExtendedBalance;
 			roll_to(15);
 			assert_eq!(MultiPhase::current_phase(), Phase::Signed);
 
-			let (solution, _) = MultiPhase::mine_solution(2).unwrap();
+			frame_support::parameter_types! {
```

Comment:
Let's make `mine_solution` use `T::Solver` for simplicity, and instead provider a `fn set_balancing` that updates `RandomBalancing`